docs/getting-started/v3-migration.md

```diff
@@ -81,6 +81,10 @@ Chart.js is no longer providing the `Chart.bundle.js` and `Chart.bundle.min.js`.
 * `afterBuildTicks` now has no parameters like the other callbacks
 * `convertTicksToLabels` was renamed to `generateTickLabels`. It is now expected to set the label property on the ticks given as input
 
+#### Scales
```

Comment:
Maybe the `Ticks` and `Time Scale` sections should be sub-sections of this new `Scales` section?

Reply:
done